BaragonService/src/main/java/com/hubspot/baragon/service/managers/ElbManager.java

```diff
@@ -182,9 +182,14 @@ public void syncAll() {
     }
   }
 
-  private List<LoadBalancerDescription> elbsForGroup(BaragonGroup group) {
-    DescribeLoadBalancersRequest request = new DescribeLoadBalancersRequest(new ArrayList<>(group.getSources()));
-    return elbClient.describeLoadBalancers(request).getLoadBalancerDescriptions();
+  private List<LoadBalancerDescription> getElbsForGroup(List<LoadBalancerDescription> elbs, BaragonGroup group) {
+    List<LoadBalancerDescription> elbsForGroup = new ArrayList<>();
+    for (LoadBalancerDescription elb : elbs) {
+      if (group.getSources().contains(elb.getLoadBalancerName())) {
```

Comment:
any chance we'll get false positives with `contains()`? is there anything more explicit we can do here?